### DISCUSSION TITLE
Products Endpoint

### DISCUSSION THREAD

**Discussion by zswilliams** [Sellbrite/Sellbrite-API#23] [Q&A]
The Products endpoint documentation states that you can send an array of sku's to get back multiple skus in a query.  However, when sending a URL string like the following:

https://api.sellbrite.com/v1/products?skus=SKU1&skus=SKU2

Only the last element in the list has data returned for it.

Formatting using ?skus=SKU1,SKU2 doesn't function at all, so presumably that is not the correct formatting. 

Is there a query issue, or potentially an API issue, not behaving as expected?

**Comment by Taegost** [comment]
Looking at the documentation [here](https://developer.sellbrite.com/endpoints/products), you would pass in the array of skus as a JSON query parameter, not as a URL parameter. You would do a GET call to the endpoint `https://api.sellbrite.com/v1/products` and then pass in a JSON object with the array. You *may* need to also include the `page` parameter, but I haven't tested that.
